sktree/tests/test_honest_tree.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import pytest
+from sklearn_fork import datasets
+from sklearn_fork.metrics import accuracy_score
+
+from sktree.tree import HonestTreeClassifier
+
+# also load the iris dataset
+# and randomly permute it
+iris = datasets.load_iris()
+rng = np.random.RandomState(1)
+perm = rng.permutation(iris.target.size)
+iris.data = iris.data[perm]
+iris.target = iris.target[perm]
+
+
+@pytest.mark.parametrize("criterion", ["gini", "entropy"])
+@pytest.mark.parametrize("max_features", [None, 2])
+def test_iris(criterion, max_features):
```

Comment:
Can we test various trees as input to `HonestTreeClassifier`? 

E.g.

HonestTreeClassifier(ObliqueDecisionTree)
HonestTreeClassifier(PatchObliqueDecisionTree)